src/router/Router.tsx

```diff
@@ -52,6 +52,16 @@ export const RoutesProvider = () => {
       <Route path='/'>
         <Route element={<Layout />}>
           {home}
+          <Route
+            path='processes/:id/:pkey'
+            element={
+              <SuspenseLoader>
+                <Process />
+              </SuspenseLoader>
+            }
+            loader={async ({ params }) => client.fetchElection(params.id)}
+            errorElement={<Error />}
+          />
```

Comment:
As I said yesterday, I don't think we should be using the router to define a specific route, since doing it this way, all private keys will be logged in the backend servers where this app is deployed. I'd say to use the hash part of the route.